cics-bundle-maven-plugin/src/main/java/com/ibm/cics/cbmp/BuildCICSBundleMojo.java

```diff
@@ -130,6 +130,52 @@ private boolean shouldBuild() {
 				buildContext.hasDelta("pom.xml");
 	}
 
+	private List<Define> copyBundlePartsFromResources() {
+		List<Define> defines = new ArrayList<Define>();
+		// TODO can we replace this with a better symbolic so the user can redefine their resources dir location?
```

Comment:
I think this is a pretty good idea.  Should be an `@Parameter`, probably with user-property defined, so that the scan directory can be overridden.  This keeps options open for doing bundle generation in `war` etc packaged projects.  I'll try and put together an issue with some sample config for how I think that might work.  There's definitely always going to be a case for having stand-alone cics-bundle modules though.